lib/config/pricing/src/index.js

```diff
@@ -148,7 +151,7 @@ const id = (oid, rtype, ppid, time, auth, cb) => {
   if(!rtype) {
     debug('Undefined resource type');
     return cb(undefined,
-      businessError(true, 'Pricing config - Undefined resource type'))
+      businessError('Pricing config - Undefined resource type'))
```

Comment:
I agree with the first point where we should stop the execution of the flow once there is an error in step 1 / 2. 

For the second point: in the pull other pull request that I want to get reviewed, I posted a similar question. I only made the changes in the mapper mainly because it doesn't pick up previous accumulated/aggregated usage and adds more usage on top of it.

In the case of reducer, if aggregator fails, aggregator needs to tell accumulator not to post the updated accumulated usage to the output db. Although I'm not too sure what kind of business error can happen in aggregator, but not in accumulator (Both are grabbing the same metering/rating plan).

Thanks. I'll made changes based on the first point for now!